wardrobe/kitties/src/lib.rs

```diff
@@ -165,6 +207,7 @@ pub struct KittyData {
     pub free_breedings: u64, // Ignore in breed for money case
     pub dna: KittyDNA,
     pub num_breedings: u128,
+    pub name: [u8; 4],
```

Comment:
Also, I think the fields should follow a more logical order, such as:
- DNA
- Name
- Parents
- free_breedings
- num_breedings

Reply:
Ok will update as suggested.

Reply:
> It is generally better to add new members at the end of a struct. This practice aligns with the principle of maintaining backward compatibility. 

Although you are not wrong, at this stage of the development we don't need to care about this, and we should prioritize doing stuff that makes sense and is clear and understandable.

And sometimes, you _do_ want to break compatibility.

Reply:
Ok. I changed as you suggested.